src/components/input.js

```diff
@@ -240,6 +240,14 @@ const withAutocomplete = WrappedComponent => ({
     { itemToString: v => v ? itemToString(v) : value } :
     { selectedItem: value }
 
+  const inputEl = useRef()
+  let clearMethod
```

Comment:
I would probably call this `clearFunction` or `clearFn` if I'm understanding the usage correctly. "Method" means "a function defined on an object" to me. Your call.

Reply:
This should be a ref instead of a let.

Reply:
Sorry, can you expand on what you mean by that?

Reply:
Isaac is right. `useRef` is the pattern. Take a look at `src/components/Collapse.js` for an example. It will work very similarly to what you have, but `useRef` is more idiomatic to our codebase.

Reply:
Did you mean you wanted the variable called `clearSelectionRef`? I updated the PR to do that, but I am only 20% sure that's what you guys meant.

> This should be a ref instead of a let
`let clearMethod` -> `ref clearMethod` is not a valid line of code, and that won't compile. So I'm just kind of flying off the cuff now, and I need more specificity to fix the issue.